skillmap/src/components/makecodeFrame.tsx

```diff
@@ -188,7 +192,9 @@ class MakeCodeFrameImpl extends React.Component<MakeCodeFrameProps, MakeCodeFram
             }
         };
 
-        await saveProjectAsync(project);
+        if (project.header.tutorial || project.header.tutorialCompleted) {
```

Comment:
is there a way to also check if the activity we're currently in is of "tutorial" type? otherwise maybe add a comment that this will have to be updated if/when we generalize the types of activities

Reply:
Added to the props